projects/fusion-ui/components/chart/v4/stories/chart-v4-story-wrapper.component.ts

```diff
@@ -35,6 +39,7 @@ export class ChartV4WrapperComponent {
     @Input() data: ChartData;
     @Input() type: ChartType;
     @Input() options: any;
+    @Input() testId: string = 'charts-default';
```

Comment:
Are you sure that here need default testid if you will use this value from URL parameter?